src/ui/src/js/run.js

```diff
@@ -115,17 +115,22 @@ export default function appRun(
     // SystemService.loadSystems();
 
     $rootScope.loadUser(token).then(() => {
-      $state.reload();
```

Comment:
This wasn't strictly related to this PR, but as far as I could tell this wasn't a valid statement and only resulted in an error getting spit out on the console.  I got annoyed at seeing the error so I just took this line out.